pkg/apis/extensions/v1alpha1/types_worker.go

```diff
@@ -128,6 +128,8 @@ type WorkerPool struct {
 	// Zones contains information about availability zones for this worker pool.
 	// +optional
 	Zones []string `json:"zones,omitempty"`
+	// AllowSystemComponents determines whether the pool should be allowed to host system components or not (defaults to true)
```

Comment:
```suggestion
	// AllowSystemComponents determines whether the pool should be allowed to host system components or not (defaults to true)
	// +optional
```

Reply:
Removed due to the comment from @tim-ebert below: https://github.com/gardener/gardener/pull/2480#discussion_r443415041